### ISSUE TITLE
Circle annotations: incorrect radius size

### ISSUE THREAD

**Issue by chpicone** [geosolutions-it/MapStore-C179#14]
When we create an annotation defining a radius value in meters we obtain a circle with different size when we measure it using the measure tool.
In this example we created a circle with 100 meters radius.
When we measure the diameter we were expecting 200 m (more or less) instead off 144 m:

![image.png](https://images.zenhubusercontent.com/5e98219c18069195a59f5217/a550521d-1cfa-4e4f-b691-d8866493665b)

**Comment by chpicone** [comment]
As our discussion in skype, can someone take a look on this ticket?
It is important to have just this fix in the MapStore-C179 repo to avoid to update to a newer version.

**Comment by tdipisa** [comment]
@chiaracurcio @simboss 
I've opened an issue in MS to improve the current behavior (https://github.com/geosolutions-it/MapStore2/issues/6503).

This MS project is currently using an [old MS revision](https://github.com/geosolutions-it/MapStore2/commits/d8ae14b04180cd5afed2976fb1e60f5c787d4fc4) that  corresponds to v2020.01.01 where all latest updates for the annotation tool (released in v2021.01.00) are obviously missing.

If it is not possible to update the client instance to the new MS version, as soon as we have implemented the enhancement in master (~2pts), we will have to backport it to the closest stable branch ([2020.01.xx](https://github.com/geosolutions-it/MapStore2/commits/2020.01.xx)). This may require more or less similar effort (maybe less) as the entire annotation tool has recently been redesigned.

If it is ok for you we could start the development next monday, Feb 8 and put this enhancement also in the next minor release of MS.

**Comment by simboss** [comment]
the sooner, the better...

I would not mind fixing this directly for the client and then port to master. Actually, I sontrgly advise that 
they have been waiting enough already...

**Comment by tdipisa** [comment]
@simboss, of course. We can do the first development in the old stable branch and then move it to the master one later.

**Comment by chpicone** [comment]
@tdipisa can I consider to be able to install this new version at 10th/11th February?

**Comment by tdipisa** [comment]
@chpicone yes.

**Comment by tdipisa** [comment]
@chpicone we have updated the master branch through [this PR](https://github.com/geosolutions-it/MapStore-C179/pull/15). Please let me know if all is working properly for you too.

**Comment by chpicone** [comment]
@tdipisa it works as expected:
![image](https://user-images.githubusercontent.com/2872823/107356439-1a7c6080-6ad1-11eb-8ec8-29a73cfdce2d.png)

**Comment by tdipisa** [comment]
@chpicone good. I will ask @dsuren1 tu migrate this to MS master as soon as possible.
